File: Python/1044.py
# -*- coding: utf-8 -*-
n1, n2 = input().split(" ")
n1 = int(n1)
n2 = int(n2)

if n2 % n1 == 0 or n1 % n2 == 0:
    print("Sao Multiplos")

else:
    print("Nao sao Multiplos")

File: Java/1044.java
import java.io.IOException;
import java.util.Scanner;

public class Main {
 
    public static void main(String[] args) throws IOException {
        Scanner leitor = new Scanner(System.in);
        
        int n1 = leitor.nextInt();
        int n2 = leitor.nextInt();
        
        if (n2 % n1 == 0 || n1 % n2 == 0) {
            System.out.println("Sao Multiplos");
        }
        else {
            System.out.println("Nao sao Multiplos");
        }
    }
 
}